backend/frontend/views.py

```diff
@@ -616,6 +617,17 @@ def view_partial_teams_list(request):
             user=user, perms=Team.PERMISSION_VIEW, klass=all_teams, any_perm=True
         )
 
+    if request.method == "POST":
+        form = SearchTeamForm(request.POST)
+
+        if form.is_valid():
+            search_term = form.cleaned_data["search_term"]
+
+            teams = Team.objects.filter(
+                active=True, name__istartswith=search_term
```

Comment:
Why not icontains?
And what if there are multiple words in the search term?

If there are multiple words then we should check if the name contains any of them. So we would be using Q and |